src/bin/pg_dump/pg_dump.c

```diff
@@ -13193,6 +13193,9 @@ dumpOpclass(Archive *fout, const OpclassInfo *opcinfo)
 
 	PQclear(res);
 
+	if(isBabelfishDatabase(fout))
+		babelfishDumpOpclassHelper(fout, opcinfo, &q, &needComma);
```

Comment:
Don't need to pass the address of `q`, pass it directly instead.

Reply:
done